### DISCUSSION TITLE
[Clarity] Improve wording around "variation" bank tags

### DISCUSSION THREAD

**Discussion by izinman** [runelite/runelite#18251] [Ideas]
Simply put, the current wording for "variation" bank tags does not do a good job of explaining this useful feature. I've used bank tags extensively and only after coming to suggest a feature for variations did I realize that it's already implemented, just poorly explained. 

I would like to propose a very simple, but helpful change. 

Current wording for the edit tag interface:
"(append * for variation tag)"

Proposed change:
"(append * to apply the tag to variations of this item, e.g. potion doses, jewelry charges, etc)"

It seems to me that there is plenty of screen real estate to fit the longer text. Further, this helpful but admittedly wordy prompt text could optionally be applied only if a user currently has no variation tags in use, then collapsed to the existing text once at least a single variation tag is applied.

**Comment by Alexsuperfly** [comment]
a more extensive explanation of variations exists in the wiki entry for the plugin, which you can find by right clicking the plugin

**Reply by izinman** [reply]
Sure. Is that a good reason not to make the in-game hint more clear? Most users will read what's in front of them, rather than RTFM.

**Reply by izinman** [reply]
I would argue that given that bank tags are enabled by default, that reduces the likelihood of a user clicking through plugin config to read the wiki and fully understand it. Many first-time Runelite users may not even realize that bank tags are a Runelite plugin and not part of the base game.

**Reply by izinman** [reply]
Don't want to bother, but do you have any more thoughts on this? Implementation-wise it's an effortless 1-line string change, and I think it would be a nice help for players that are new to bank tags.
